e2e/cypress/integration/messaging/collapse_link_spec.js

```diff
@@ -0,0 +1,99 @@
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Messaging', () => {
+    before(() => {
+        // # Set the configuration on Link Previews
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                EnableLinkPreviews: true,
+            },
+        });
+
+        cy.apiLogin('user-1');
+        cy.apiSaveShowPreviewPreference();
+        cy.apiSavePreviewCollapsedPreference('false');
+
+        cy.apiLogin('sysadmin');
+        cy.apiSaveShowPreviewPreference();
+        cy.apiSavePreviewCollapsedPreference('false');
+
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M18708-Link preview - Removing it from my view removes it from other user\'s view', () => {
+        const message = 'https://www.bbc.com/news/uk-wales-45142614';
+
+        // # Create new DM channel with user's email
+        cy.visit('/ad-1/channels/town-square');
+
+        // # Post message to use
+        cy.postMessage(message);
+
+        cy.getLastPostId().then((postId) => {
+            // * Check the preview is shown
+            cy.get(`#${postId}_message`).find('.attachment--opengraph').should('exist');
+            cy.get(`#${postId}_message`).find('.attachment__image').should('exist');
+
+            // # Log-in to the other user
+            cy.apiLogin('sysadmin');
+            cy.visit('/ad-1/channels/town-square');
+
+            // * Check the preview is shown
+            cy.get(`#${postId}_message`).find('.attachment--opengraph').should('exist');
+            cy.get(`#${postId}_message`).find('.attachment__image').should('exist');
+
+            // # Log-in back to the first user
+            cy.apiLogin('user-1');
+            cy.visit('/ad-1/channels/town-square');
+
+            // # Collapse the preview
+            cy.get(`#${postId}_message`).find('.post__embed-visibility').click({force: true});
+
+            // * Check the preview is not shown
+            cy.get(`#${postId}_message`).find('.attachment--opengraph').should('exist');
+            cy.get(`#${postId}_message`).find('.attachment__image').should('not.exist');
+
+            // # Log-in to the other user
+            cy.apiLogin('sysadmin');
+            cy.visit('/ad-1/channels/town-square');
+
+            // * Check the preview is shown
+            cy.get(`#${postId}_message`).find('.attachment--opengraph').should('exist');
+            cy.get(`#${postId}_message`).find('.attachment__image').should('exist');
+
+            // # Log-in back to the first user
+            cy.apiLogin('user-1');
+            cy.visit('/ad-1/channels/town-square');
+
+            // * Check the preview is shown
+            cy.get(`#${postId}_message`).find('.attachment--opengraph').should('exist');
+            cy.get(`#${postId}_message`).find('.attachment__image').should('exist');
```

Comment:
Hi @larkox 
As mentioned by @saturninoabril, Cypress is not consistent in storing the state of the expand/collapse images. Hence it has failed in one of the runs in my case. Can you remove this step as it's not necessary to validate. 

<img width="1650" alt="Screenshot 2019-11-15 at 6 24 03 PM" src="https://user-images.githubusercontent.com/1429138/68944930-2184c800-07d5-11ea-9983-4ef1df14035a.png">